index.js

```diff
@@ -9,31 +12,113 @@ function ask(questionText) {
 
 // remember the StateMachine lecture
 // https://bootcamp.burlingtoncodeacademy.com/lessons/cs/state-machines
-let states = {
-  'roomOne': { canChangeTo: [ 'roomTwo' ] },
-  'roomTwo': { canChangeTo: [ 'roomThree' ] },
-  'roomThree': { canChangeTo: [ 'roomOne' ] }
+
+let rooms = {     // was given in starter code 
+  'outside':   { 
```

Comment:
All these rooms have the same basic structure, so you could make them through a class definition with a constructor instead, which would make your life easier, and would improve the scale-ability of your program